toolchain/check/testdata/facet/min_prelude/fail_namespace_type.carbon

```diff
@@ -7,9 +7,9 @@
 //
 // AUTOUPDATE
 // TIP: To test this file alone, run:
-// TIP:   bazel test //toolchain/testing:file_test --test_arg=--file_tests=toolchain/check/testdata/facet/min_prelude/namespace_type.carbon
+// TIP:   bazel test //toolchain/testing:file_test --test_arg=--file_tests=toolchain/check/testdata/facet/min_prelude/fail_namespace_type.carbon
```

Comment:
FWIW, the filename shouldn't matter when the individual tests inside are failing

Reply:
Yeah, I just thought it's a nice signal - I don't anticipate any passing tests with namespaces as facets.